hippunfold/workflow/scripts/create_warps.py

```diff
@@ -34,7 +34,7 @@ def summary(name, array):
 coord_io = coord_io_nib.get_fdata()
 
 #get mask of coords  (note: this leaves out coord=0)
-mask = (coord_ap > 0) | (coord_pd > 0) | (coord_io > 0) # some points were lost, especially IO
+mask = (coord_ap > 0) | (coord_pd > 0) # some points were lost, especially IO
```

Comment:
does this end up reducing the number of voxels in the mask? (since it is a logical OR)?

Reply:
it actually increases the number of voxels in the mask - but only by a handful.

Reply:
ah yea, of course! -- I guess my fault for multi-tasking when I read this!